resources/quality/nwa3d_a5/nwa3d_a5_fast.inst.cfg

```
@@ -0,0 +1,124 @@
+[general]
+version = 1
```

It has to be `version = 4`
```suggestion
version = 4
```